src/frontend/src/env/networks.btc.env.ts

```diff
@@ -14,3 +14,9 @@ export const BITCOIN_CANISTER_IDS: Record<MinterCanisterIdText, CanisterIdText>
 		[IC_CKBTC_MINTER_CANISTER_ID]: 'ghsi2-tqaaa-aaaan-aaaca-cai'
 	})
 };
+
+export const BTC_MAINNET_ENABLED =
```

Comment:
This property should probably be added to `.env.example` as well?

Reply:
since it is a temporary flag to be used only in local replica until we are ok with Bitcoin features, I thought to not add it there, like we do for example for the Chain Fusion flag

but not strong opinion, if you prefer it there, I will add it